browser/js/dashboard/dashboard.controller.js

```diff
@@ -4,48 +4,33 @@ app.controller('DashboardCtrl', function($rootScope, $scope, $mdDialog, MdHelper
         $scope.draggingNow = true;
     }
 
-    var dateFilter = function(obj){
-        return _.pickBy(obj, function(snippetCreated){
-            var diff = (snippetCreated - $scope.currentWeek)/(1000*60*60*24);
-            return (diff < 7 && diff > 0);
-        })
+    $scope.dateInRange = function(date){
```

Comment:
This could be a good place for your own [custom angular filter](https://scotch.io/tutorials/building-custom-angularjs-filters).